conf/application.conf

```diff
@@ -98,12 +98,12 @@ microservice {
     }
 
     features {
+      default = disabled
       betaBanner = enabled
       changeErrorLink = enabled
-      default = disabled
       ead = enabled
       sfus = enabled
-      sfusSecureMessaging = enabled
+      secureMessaging = sfus
```

Comment:
Is this actually working? Should not `sfus` be between braces (... or something like that, I do not remember the correct syntax now)